doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md

```diff
@@ -45,8 +45,50 @@ kubectl get all -n prometheus-system
         enabled: true
         org_role: Viewer
   ```
-
-
+* In my environment (with Ray version 2.9.0+), the Ray dashboard encountered a cross-origin issue when embedding Grafana charts. This problem was resolved after modifying the grafana.ini configuration file. Below is the complete configuration file:
```

Comment:
How can I reproduce the issue in my local Kind cluster?

Reply:
@bertreyking can you reach out to me on Ray Slack workspace so that I can accelerate the review process? My Slack handle is "Kai-Hsun Chen (ray team)". The current PR has several issues:

* If I can't reproduce it, I can't verify the correctness of the PR, so I am not confident to merge this PR.
* The document asks users to install the Prometheus / Grafana via Helm chart. It would be better to update the `values.yaml`.

Reply:
> * The document asks users to install the Prometheus / Grafana via Helm chart. It would be better to update the `values.yaml`.

- Yes, this is not an issue with Ray itself; it only occurs because Ray embeds Grafana, and the problem arises when users try to view metric data in the Ray dashboard.